cluster/dune

```diff
@@ -1,26 +1,23 @@
 (library
-  (name xcp_cluster)
-  (public_name xapi-idl.cluster)
-  (modules (:standard \ cluster_cli))
-  (flags (:standard -w -39))
-  (libraries xapi-idl threads rpclib.core)
-  (wrapped false)
-  (preprocess (pps ppx_deriving_rpc))
-)
+ (name xcp_cluster)
+ (public_name xapi-idl.cluster)
+ (modules (:standard \ cluster_cli))
+ (flags (:standard -w -39))
+ (libraries xapi-idl threads rpclib.core)
+ (wrapped false)
+ (preprocess (pps ppx_deriving_rpc)))
 
 (executable
-  (name cluster_cli)
-  (modules cluster_cli)
-  (libraries
-    cmdliner
-    rpclib.cmdliner
-    rpclib.markdown
-    xapi-idl.cluster
-  )
-)
+ (modes byte exe)
```

Comment:
I haven't realized that the default here would be `byte`.

Reply:
That's what the documentation states, this was the default value for linking modes before dune 2.0.